packages/flutter/lib/src/painting/shader_warm_up.dart

```diff
@@ -56,12 +56,10 @@ abstract class ShaderWarmUp {
 
   /// The size of the warm up image.
   ///
-  /// The exact size shouldn't matter much as long as it's not too far away from
-  /// the target device's screen. 1024x1024 is a good choice as it is within an
-  /// order of magnitude of most devices.
+  /// The exact size shouldn't matter much as long as all draws are onscreen.
```

Comment:
Can you add some context here about why we think 100.0x100.0 is good?  Something like what you wrote in the response to my comment on the PR issue woul dbe good.

Reply:
Done